plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -511,8 +564,26 @@ void toIndex(FileIndex[] files, Element root, string htmlFileDir) @trusted {
                     return "background-color: lightgreen";
                 return null;
             }();
-
             r.addChild("td", format!"%.3s"(score)).style = style;
+
+            float scoreChange;
+            if(f.display in scoreDifference){
+              scoreChange = scoreDifference[f.display];
+            }else{
+              scoreChange = 0;
+            }
+
+            const scoreChangeStyle = () {
+                if (scoreChange < 0)
+                    return "background-color: salmon";
+                if (scoreChange == 0)
```

Comment:
need to use some kind of threshold because it will never be `0`.
see impl for prediction of trend.

Reply:
it may be possible, should be possible, to reuse that impl by separating it to `utility.d`.
the colors should also be possible to reuse. one color schema to rule them all

Reply:
> need to use some kind of threshold because it will never be `0`. see impl for prediction of trend.

Currently it will be zero is there is no change in score in the last seven days, should it not be shown at all if it is zero instead?